sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-servicebus-messaging/README.md

```diff
@@ -0,0 +1,80 @@
+# Spring Cloud Azure Service Bus Messaging Sample shared library for Java
+
+## Key concepts
+
+This code sample demonstrates how to use [AzureMessageListener.java][annotation-azure-message-listener] to listen to messages from Service Bus Topic.
+
+## Getting started
+
+Running this sample will be charged by Azure. You can check the usage and bill at 
+[this link][azure-account].
+
+### Prerequisites
+- [Environment checklist][environment_checklist]
+
+### Create Azure resources
+
+1.  Create [Azure Service Bus Namespace][create-service-bus-namespace].
+    Please note `Basic` tier is unsupported.
+    
+1.  Create [Azure Service Bus Topic][create-service-bus-topic] and named `topic`. After creating the Azure Service Bus Topic,
+    you can create the subscription [Azure Service Bus Topic subscription][create-subscription] to the topic and named `sub` .
+
+## Dependency
+Because dependency `azure-spring-cloud-starter-servicebus` does not introduce the dependency about messaging, we need to add
+dependency `azure-spring-cloud-messaging`.
+
+```
+    <dependency>
+      <groupId>com.azure.spring</groupId>
+      <artifactId>azure-spring-cloud-starter-servicebus</artifactId>
+      <version>2.3.0</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.azure.spring</groupId>
+      <artifactId>azure-spring-cloud-messaging</artifactId>
+      <version>2.3.0</version>
+    </dependency>
+```
```

Comment:
Do we need to add the version tag here, so the update version script could pick up these versions.

Reply:
Just like this? ` <version>2.3.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-messaging;current} -->`